packages/components-react/card-react/src/index.test.tsx

```diff
@@ -0,0 +1,137 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import * as React from 'react';
+import { Card } from './index';
+import '@testing-library/jest-dom';
+
+describe('Card', () => {
+  it('renders an Card component', () => {
+    const { container } = render(<Card heading={'This is a Card component title'} />);
+
+    const card = container.querySelector('div:only-child');
+
+    expect(card).toBeInTheDocument();
+  });
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<Card heading="Card title" />);
+    const card = container.querySelector('div:only-child');
```

Comment:
Je test nu twee dingen tegelijk: of het een `div` rendert en of het een class name rendert. Die check op de `div` zou een losse unit test kunnen zijn, zodat je duidelijkere failures krijgt.

```suggestion
    const card = container.querySelector(':only-child');
```